src/plugins/auto/multi_schedule.hpp

```diff
@@ -16,12 +16,14 @@
 
 namespace MultiDevicePlugin {
 struct ThisRequestExecutor : public IE::ITaskExecutor {
-    explicit ThisRequestExecutor(WorkerInferRequest** ptr): _workptrptr{ptr} {}
+    explicit ThisRequestExecutor(WorkerInferRequest** ptr, MultiImmediateExecutor::Ptr test = nullptr): _workptrptr{ptr}, _testexecutor(test) {}
```

Comment:
change all these variables start with test to some other meaningful name, like _fallbackExecutor, _fallbackExec